### ISSUE TITLE
Generator bug when encoding with a to-many include path that is empty

### ISSUE THREAD

**Issue by lindyhopchris** [laravel-json-api/laravel#11]
When including a to-many relationship that is empty, we're encountering a bug in the underlying encoder:

```
Exception: Cannot traverse an already closed generator
src/Parser/RelationshipData/RelationshipDataIsCollection.php:132
```

Have opened the issue and submitted a PR here:
https://github.com/neomerx/json-api/issues/252

**Comment by lindyhopchris** [comment]
No reply to the issue raised in the Neomerx library. For the moment, I've updated our `laravel-json-api/encoder-neomerx` package to require the branch that has the fix on it.

If we don't hear back from Neomerx about the bug, we're going to have to publish our own fork. I've already forked the library here as it's critical to the Laravel JSON:API implementation, so makes sense to have a fork copy:
https://github.com/laravel-json-api/neomerx-json-api
